src/app/page/component/page/page-helpers.ts

```diff
@@ -0,0 +1,17 @@
+export function shouldShowBackButton(page: {
+  parentPage?: { position?: any };
+}): boolean {
+  return !!page.parentPage?.position;
```

Comment:
Is it OK that this will return false if `page.parentPage.position === 0`? It's slightly different from the `page.parentPage?.position !== undefined` check in `navigateBackIfPossible`. I think it would be good to use the same check unless there's a reason for them to be different.